applicationconfig/client/src/main/java/com/azure/applicationconfig/models/ConfigurationSetting.java

```diff
@@ -12,6 +12,12 @@
  * To explicitly reference null label use "\0" (url encoded as %00).
  */
 public class ConfigurationSetting {
+    /**
+     * The default label for configuration settings is the null label, "\0".
+     * This gets URL encoded as "%00".
+     */
+    public static final String NULL_LABEL = "\0";
```

Comment:
This is only used when setting filters, should we move it there?

Reply:
We also use it in our ConfigurationClient when "null" is passed in as a label: https://github.com/Azure/azure-sdk-for-java/blob/azconfig/applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java#L251